app/index.js

```diff
@@ -14,6 +15,37 @@ var paths = {
   'gulpfile.js': {}
 };
 
+function logWarning(message) {
+  var warningSign = clc.bold.cyan('<' + clc.bold.redBright('!') + '> ');
+  console.log(warningSign + clc.cyanBright(message));
+}
+
+function outputWarning() {
+  var gulp = clc.magenta('gulp'),
+      prompt = clc.cyan('$ '),
+      npm = clc.magenta('npm'),
+      quicksip = clc.yellow('quick-sip'),
+      command = clc.redBright,
+      warn = clc.cyanBright;
+  logWarning('We have installed ' + gulp + ' locally to build your-torso-project.\n');
+  console.log(warn('To use the gulp tasks setup by ' + quicksip + ', you will need to use the ' + npm + ' run cli with the following command:\n'));
+  console.log(command(prompt + npm + ' run gulp ' + clc.bold.white('<task>')));
+  console.log();
+  console.log(warn('To remove ' + gulp + ' locally and setup ' + gulp + ' globally, run the following command:\n'));
+  console.log(command(prompt + npm + ' uninstall gulp ' +clc.white('&& ') + npm + ' install -g gulp'));
+}
+
+function installLocalGulpAndBuild(generator) {
+  generator.spawnCommand('npm', ['install', 'gulp', '--save-dev'])
+    .on('exit', function() {
+      logWarning('gulp installed locally');
+      generator.spawnCommand('npm', ['run', 'gulp', 'build'])
```

Comment:
You use "npm run gulp build" here and "gulp build" below.  You should be able to use the npm run command in both cases - and pull it out to a helper fxn to share the code.